Makefile

```diff
@@ -44,7 +44,7 @@ vdi: img
 qcow2: img
 	qemu-img convert foxos.img -O qcow2 foxos.qcow2
 
-run: img ./tmp/ovmf
+run: ./tmp/ovmf
```

Comment:
@Glowman554 do you want to do this? I'm fine with that.

Reply:
That's only that he could use the make command to run it on Mac we should keep it for everybody else

Reply:
Well actually. Run is the same on all platforms. Maybe just make them 2 different commands.